src/reducers/patients.js

```diff
@@ -0,0 +1,13 @@
+import { FETCH_PATIENTS_FULFILLED } from '../actions/types';
+
+export default function(state = {patientEntries: [], meta:{}}, action)  {
+  switch (action.type) {
+    case FETCH_PATIENTS_FULFILLED:
+      const patientEntries = action.payload.data.entry.map((e) => e.resource);
+      const { total, link } = action.payload.data;
+      const meta = { total,link };
```

Comment:
We don't need `meta` here as we only use it for the total which can be derived from the length of `patients`.